docs/self-hosting.md

```diff
@@ -60,9 +75,19 @@ To mount the `daemon-config.toml` file when `docker run`'ing the image a [docker
 
 The `miningpool-observer-web` web-server and reads data from the database and generates HTML pages with the data and site templates.
 
+Optionally (if it's a different user) clone the repository, checkout a release tag and run:
+
+```sh
+cargo build --release --bin miningpool-observer-web
+```
+
+To see if it works, run `target/release/miningpool-observer-web`, but first see Configuration below.
```

Comment:
I found that I had to use an absolute path for `www_dir_path` rather than just `./wwww`, because otherwise it complained about not finding `index.html`. Didn't mention that here, because maybe I did something wrong on my end.

Reply:
I'll you deal with that; fixing the code to be consistent with how it founds the configuration file seems better than updating the docs, both both work for me.

Reply:
Seems like dropping the `./` from `./www` in the example did the trick.

Reply:
https://github.com/0xB10C/miningpool-observer/pull/46